moveit_kinematics/ikfast_kinematics_plugin/scripts/create_ikfast_moveit_plugin.py

```diff
@@ -50,284 +53,294 @@
 import yaml
 from lxml import etree
 import shutil
+import argparse
 
 plugin_gen_pkg = 'moveit_kinematics'  # package containing this file
 plugin_sub_dir = 'ikfast_kinematics_plugin' # sub directory which contains the template directory
 # Allowed search modes, see SEARCH_MODE enum in template file
 search_modes = ['OPTIMIZE_MAX_JOINT', 'OPTIMIZE_FREE_JOINT' ]
 
+
+def createParser():
```

Comment:
inconsistent function names - others are under_score. which is correct for ROS python?